packages/now-cli/src/commands/deploy/legacy.ts

```diff
@@ -259,7 +259,7 @@ export default async function main(
   quiet = !isTTY;
   ({ log, error, note, debug, warn } = output);
 
-  const infoUrl = 'https://zeit.co/guides/migrate-to-zeit-now';
+  const infoUrl = 'https://vercel.com/guides/migrate-to-zeit-now';
```

Comment:
```suggestion
  const infoUrl = 'https://vercel.com/guides/migrate-to-vercel';
```

Reply:
Fixed this and all other links to guides and docs